### PR TITLE
BaSP-MR-03: Admins put and delete

### PR DESCRIPTION
**Pull request by francoax**
Admins development:
- Schema 
- Update validations as middleware
- Put and delete endpoints

### CODE REVIEW

File: src/controllers/admins.js [hunk 1]
```diff
@@ -10,9 +10,41 @@ const updateAdmin = (req, res) => {
...
-    firstName, lastName, dni, phone, email, city, password,
+    firstName,
+    lastName,
+    dni,
+    phone,
+    email,
+    city,
+    password,
   } = req.body;
+
```
[the remark this answers is not in the document]
Comment (IgnacioGalceran) [reply src/controllers/admins.js:23]: I agree with Ivan, you should also verify that the email and the ID number are not from the current ID you are updating.

[the remark this answers is not in the document]
Comment (IgnacioGalceran) [reply src/controllers/admins.js:23]: you can do this, one for the email and another for dni
`Admins.findOne({ email }).then((repeatedEmail) => {
if (repeatedEmail && repeatedEmail.id !== req.params.id)`
}

> I agree with Ivan, you should also verify that the email and the dni are not from the current ID you are updating.

[the remark this answers is not in the document]
Comment (francoax) [reply src/controllers/admins.js:23]: @IgnacioGalceran I can't follow what you are saying, but I did the validation if there is another admin with that dni or email.  I'll commit the changes in a moment

[the remark this answers is not in the document]
Comment (IgnacioGalceran) [reply src/controllers/admins.js:23]: i see, thats another way to fix it! well done

**Review by nailaghigo — APPROVED** [review]

**Pull request merged**